lib/common/common/src/math.rs

```diff
@@ -16,3 +16,51 @@ pub fn fast_sigmoid(x: ScoreType) -> ScoreType {
 pub fn scaled_fast_sigmoid(x: ScoreType) -> ScoreType {
     0.5 * (fast_sigmoid(x) + 1.0)
 }
+
+/// Sample size for an unlimited population, confidence level 0.99 and margin of error 0.01.
+pub const SAMPLE_SIZE_CL99_ME01: usize = 16641;
+
+/// Calculates the ideal sample size for a given population size and proportion
+/// at confidence level 0.99 and margin of error 0.01.
+///
+/// If population is `None`, considers an unlimited population.
+pub fn ideal_sample_size(population_size: Option<usize>) -> usize {
```

Comment:
Is this function used?

Reply:
Well, not actually but I didn't want to drop the implementation in case we would need it later